nisdc/nisemidevicecontrol.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, session_name, resource_name, grpc_channel) :
 class SemiconductorDeviceControl:
     """This class is used for Instrument Studio export configuration."""
 
-    def __init__(self, isconfigpath):
+    def __init__(self, isconfigpath=None):
```

Comment:
If we are going with this fix, then we need to provide a proper documentation and example how to use this wisely

Reply:
where should the documentation be added?